apps/CMakeLists.txt

```diff
@@ -25,29 +25,29 @@ add_subdirectory(helpers)
 GRCC_COMPILE(grgsm_livemon)
 GRCC_COMPILE(grgsm_livemon_headless)
 
-set(grgsm_flowgraphs "")
-OPTION(ENABLE_GRCC "Compile the flowgraphs with grcc" ON)
-OPTION(ENABLE_GRGSM_LIVEMON "Compile grgsm_livemon" ON)
-OPTION(ENABLE_GRGSM_LIVEMON_HEADLESS "Compile grgsm_livemon_headless" ON)
+set(gsm_flowgraphs "")
+#OPTION(ENABLE_GRCC "Compile the flowgraphs with grcc" ON)
```

Comment:
Hi @bkerler,

Why did you commented out `ENABLE_GRCC` option? This seems to break generation of `grgsm_livemon` and `grgsm_livemon_headless` as reported in the gr-gsm's mailing list.

https://groups.google.com/g/gr-gsm/c/F1zUJz2DVEc/m/w3m85luRIQAJ

Reply:
I uncommented it because otherwise the package fails to compile due to gnuradio 3.9 package compiling changes. So far I couldn't figure out how to fix it as it refers to packages to import that can't be imported because they aren't compiled.

Reply:
Can you post the exact error messages and list the packages that aren't compiled?